gaia-component.js

```diff
@@ -147,6 +83,26 @@ var base = {
     },
 
     /**
+     * A convenient method for triggering l10n for component's shadow DOM.
+     */
+    setupShadowL10n: function() {
+      if (!document.l10n) { return; }
```

Comment:
If the user of the component is not using `l10n.js` then their `.localizeShadow()` function will never be called. Perhaps this is OK for a V1?